### ISSUE TITLE
Add option for miles instead of kilometers

### ISSUE THREAD

**Issue by AZJerry** [ge0rg/aprsdroid#200]
Please an an option to show distances in miles. For those of us in the USA it would be greatly appreciated.

**Comment by tylojuky** [comment]
I was just about to submit the same request...

Yes - please give us yanks some miles, feet, mph, etc.

**Comment by kevinhooke** [comment]
Searching online to see how to configure to display miles and found this request, I guess this is still not added?

**Comment by penguin359** [comment]
No, I just checked and it does not appear to be available yet. I can try working on it in a few days when I have a little more free time.

**Comment by na7q** [comment]
Option for Miles or Kilometers has been coded.
https://github.com/na7q/aprsdroid/commit/a8c771f0d26978c4474f2b2be7452d678e42aadb

**Comment by lgwapnitsky** [comment]
When will it be released?

**Comment by na7q** [comment]
> When will it be released?

long ago on my releases. 
https://na7q.com/aprsdroid-osm/
